README.md

```diff
@@ -1,2 +1,51 @@
 # robotont_demos
 This repository is a ROS package that contains various demos showing the capabilities of the Robotont platform
+
+## Setting up the connection between the robot and your pc
+### 1. Connect your pc to Robotont's hotspot or the same network Robotont is connected to.
+### 2. Configure Robotont's side of the network:
+* Open a new terminal window on your pc
+* Estabish an SSH connection to the robot:<br/>
+```ssh username@robotont_IP_address```
+* If the computer asks a "yes/no" question, type "yes"
+* Enter the password: t0ndik00bas
+* Set ROS_MASTER_URI: <br/>
+```export ROS_MASTER_URI=http://robotont_IP_address:11311```
+* Set ROS_HOSTNAME: <br/>
+```export ROS_HOSTNAME=robotont_IP_address```
+
+### 3. Configure the pc's side of the network:
+* Open a new terminal window on your pc
+* Set ROS_MASTER_URI: <br/>
+```export ROS_MASTER_URI=http://robotont_IP_address:11311```
+* Set ROS_HOSTNAME: <br/>
+```export ROS_HOSTNAME=your_pc_IP_address```
+
+## 3D mapping
```

Comment:
Siin ja edaspidi. Iga demo puhul on prequisites (ehk ühekordne asi) ja igakorne jooksutamine. Ma jagaks seeõttu iga demo alapeatüki kahte ossa. Samuti tuleb selgelt välja kirjutada, millise arvuti peal konkreetset käsku jooksutada. Näiteks:
**3D mapping**
**_Setup_**
_you only need to run these commands if this is the first time you you run this demo with the current user PC or ROBOTONT on-board computer_
On ROBOTONT on-board computer, install ROS wrapper of RTAB-Map
`sudo apt-get install ros-melodic-rtabmap-ros`

**_Launching the demo_**
On ROBOTONT on-board computer, launch 3d_mapping.launch:
`roslaunch robotont_demos 3d_mapping.launch`

On user PC, launch display_3d_mapping.launch to visualize the result:
`roslaunch robotont_demos display_3d_mapping.launch`

Reply:
ja tegelikult peab enne sudo aptiga pigaldamist olema käsk `sudo apt update`

Reply:
ja läbivalt võiks apt-get asemel kasutada lihtsalt `apt`.